evm_arithmetization/tests/empty_tables.rs

```diff
@@ -58,6 +61,34 @@ fn empty_tables() -> anyhow::Result<()> {
             &config,
         )
     );
+
+    {
+        let gate_serializer = DefaultGateSerializer;
+        let generator_serializer = DefaultGeneratorSerializer::<C, D>::default();
+
+        let timing = TimingTree::new("serialize AllRecursiveCircuits", log::Level::Info);
```

Comment:
Currently the output is like this, and we do not want the total time includes the duration of serialize/deserialize functions.

```assembly
[2024-10-17T02:50:26Z INFO  plonky2::util::timing] 2.4075s to serialize AllRecursiveCircuits
[2024-10-17T02:50:26Z INFO  empty_tables] AllRecursiveCircuits length: 4007472777 bytes
[2024-10-17T02:50:30Z INFO  plonky2::util::timing] 3.5988s to deserialize AllRecursiveCircuits
[2024-10-17T02:50:58Z INFO  plonky2::util::timing] 53.6965s to Empty Table Test
[2024-10-17T02:50:58Z INFO  plonky2::util::timing] | 2.3495s to Create all STARK proofs
[2024-10-17T02:50:58Z INFO  plonky2::util::timing] | 16.4366s to Create all recursive circuits
[2024-10-17T02:50:58Z INFO  plonky2::util::timing] | 7.9814s to Prove segment
[2024-10-17T02:50:58Z INFO  plonky2::util::timing] | 0.0034s to Verify segment proof
``` 

How about using the root TimingTree object instead? Or move this segment below the root TimingTree's print()?

Reply:
I moved it below!